library/src/main/java/me/toptas/fancyshowcase/FancyImageView.java

```diff
@@ -127,15 +151,17 @@ private void drawCircle(Canvas canvas) {
      * @param canvas canvas to draw
      */
     private void drawRoundedRectangle(Canvas canvas) {
-        float width = mCalculator.roundRectWidth();
         float left = mCalculator.roundRectLeft();
```

Comment:
Thisr rounded rectangle animation moves only up and down sides. But it should also animate to left and right sides. 
`mCalculator.roundRectLeft()` and `mCalculator.roundRectRight()` methods should get `mAnimCounter`, `mAnimMoveFactor` parameters to animate to  left/right sides.